agent/bench-scripts/pbench-fio.md

```diff
@@ -339,3 +339,27 @@ Here are the pbench-fio parameters:
 * **--numjobs=uint** - the total number of "jobs" to run.  This is typically set to the number of clients above.  Typically in the job file, you use numjobs=1.
 * **--pre-iteration-script=path** - this points to a local executable script (doesn't have to be bash) that will execute before each sample is run.  For an example of how this can be useful, see cache-dropping section above.
 * **--max-stddev=uint** - maximum percent deviation (100.0 \* standard deviation / mean) allowed for a successful test.  If you get test failures, consider either lengthening your test, changing test procedure to improve repeatability, or increasing this parameter.  Default is 5%.
+
+
+## firewalls
+
+*You must ensure that the network firewall is not up, or poke holes in the firewall
+for pbench-fio when using remote clients*.  Typically there are two possible
+firewall implementations encountered:
+
+* the **firewalld** service
+* the **iptables** service
+
+To temporarily disable (this may give security folks heartburn):
+
+    # systemctl stop firewalld
+    # systemctl stop iptables
+
+To temporarily enable port under firewalld use:
+
+    # firewall-cmd --add-port=8765/tcp
+
+Where "8765" is the default port pbench-fio will use for the fio clients.
+If you are using multiple clients, without specifying explicit ports, then
+starting with port 8765, pbench-fio will use sequential ports for each client.
```

Comment:
Adjusted the text to include the use of `--unique-ports`, but not sure we want to change the port numbers used with this commit.

It is `fio` the has a default client port of `8765`, so that is pretty ingrained in the usage pattern today.

Reply:
My real concern for this comment was the accuracy of the text wrt when it uses sequential ports (and, I presume you've addressed that).

I'm less concerned about using a specific port (like 8765) than I am about the sequential usage covering a substantial amount of the otherwise "reserved" port range.  But, if an issue with that actually arises, I presume the user can specify `--unique-ports` together with a starting port in the unreserved range and resolve the problem.